docs/source/configuration.rst

```diff
@@ -258,14 +258,23 @@ very large difference to how the application behaves.
     more performant on underloaded dynos, so a better heuristic is ``0.25*number_of_bots``.
 
 ``dyno_type``
-    This determines how powerful the heroku dyno that's started is. It applies to both
-    web and worker dyno types. The minimum recommended is ``standard-1x``, which should be
-    sufficient for experiments that do not rely on real-time coordination, such as
-    :doc:`demos/bartlett1932/index`.
-    Experiments that require significant power to process websocket events should consider
-    the higher levels, ``standard-2x``, ``performance-m`` and ``performance-l``. In all but
-    the most intensive experiments, either ``dyno_type`` or ``num_dynos_web`` should be
-    increased, not both.
+    This determines how powerful the heroku dynos started by Dallinger are. It is applied
+    as the default for both web and worker dyno types. The minimum recommended is
+    ``standard-1x``, which should be sufficient for experiments that do not rely on
+    real-time coordination, such as :doc:`demos/bartlett1932/index`. Experiments that
+    require significant power to process websocket events should consider the higher
+    levels, ``standard-2x``, ``performance-m`` and ``performance-l``. In all but the
+    most intensive experiments, either ``dyno_type`` or ``num_dynos_web`` should be
+    increased, not both. See ``dyno_type_web`` and ``dyno_type_worker`` below
+    for information about more specific settings.
+
+``dyno_type_web``
```

Comment:
I'd spell out that this (and `dyno_type_worker`) will override the value of `dyno_type` if provided.

Reply:
Done.